### ISSUE TITLE
Does this still work?

### ISSUE THREAD

**Issue by jaymann220** [jimprintsthings/homebridge-febolink#1]
Hello, i am having issues installing this, would love some help with this, only reason i installed homebridge was for my febo fireplace link here

**Comment by jimprintsthings** [comment]
No, I started on it, and just never got into it.  Honestly I thought I had this set to private since I never started.  Sorry.   I did get my febolink going by capturing on and off commands.  I got hung up in setting the temperatures which really is why I gave up.   But I used the homebridge-cmd to call simple on/off scripts that I have.

**Comment by jaymann220** [comment]
I am just looking for an on off, I'm not worried about temp. if you still have them that would be awesome.
